link.lds

```diff
@@ -29,48 +29,57 @@ SECTIONS
 		*(.headers)
 		*(.text*)
 	}
-	. = ALIGN(64);
 	.rodata : {
 		*(SORT_BY_ALIGNMENT(.rodata*))
 	}
-	.data : {
-		*(SORT_BY_ALIGNMENT(.data*))
-	}
-	.bss : {
-		*(SORT_BY_ALIGNMENT(.bss*))
-	}
 
-	/*
-	 * Due to the 64k total size constraint, we link all page size/aligned
-	 * data together in a single section, to avoid wasting space in the
-	 * individual data/bss sections.
-	 */
-	.page_data : {
-		*(.page_data)
-	}
+	_end_of_measured = .;
 
-	.skl_info : {
-		*(.skl_info)
+	.bss : {
+		. = ALIGN(4096);
+		_bss = .;
+		/*
+		 * Due to the 64k total size constraint, we link all page size/aligned
+		 * data together in a single section, to avoid wasting space in the
+		 * individual data/bss sections.
+		 */
+		*(SORT_BY_ALIGNMENT(.bss.page_data))
+		*(SORT_BY_ALIGNMENT(.bss*))
+		. = ALIGN(4);
+		_ebss = .;
 	}
 
-	. = ALIGN(8);
-
 	/*
 	 * Bootloader must pass non-constant data (e.g. address of zeropage). Keep
 	 * it in separate section, outside of measured part of SL. This must be
 	 * done in order to keep hashes constant, it also allows us to measure SL
 	 * offline.
 	 */
+	. = ALIGN(4);
 	.bootloader_data : {
 		*(.bootloader_data)
 	}
 
+	_end = .;
+
 	/* This section is expected to be empty. */
 	.got : {
 		*(.got)
 	}
 
-	_end = .;
+	/* This section is expected to be empty, unless ld is bugged. */
+	.got.plt : {
+		*(.got.plt)
+	}
+
+	/*
+	 * This section is expected to be empty. To keep amount of data sent to the
+	 * TPM by SINIT instruction low, all data is either read-only (.rodata), or
```

Comment:
nit: s/SINIT/SKINIT/

Reply:
Fixed in https://github.com/TrenchBoot/secure-kernel-loader/compare/412bd740fd2d1b043ae6ad8a47726fdb7362848f..602004f3fab90a9502d60ef7b0ccae528c20b30f